src/Stripe.Tests.XUnit/invoices/creating_and_updating_invoices_with_manual_invoicing.cs

```diff
@@ -31,7 +32,7 @@ public creating_and_updating_invoices_with_manual_invoicing()
             var InvoiceCreateOptions = new StripeInvoiceCreateOptions
             {
                 Billing = StripeBilling.SendInvoice,
-                DaysUntilDue = 7,
+                DueDate = DateTime.Today.AddDays(7),
```

Comment:
minor but elsewhere we use `DateTime.UtcNow.AddDays(10);` (for example for RedeemBy [here](https://github.com/stripe/stripe-dotnet/blob/96cdbaffa5609ab6f8c4925342247c5a8be6f0ac/src/Stripe.net.Tests/coupons/test_data/stripe_coupon_create_options.cs#L16)).

Might be worth sticking with conventions (makes grepping for examples and similar use-cases easier)

Similarly, if you look at how `RedeemBy` is implemented, the code uses a `long?` instead

```
        public DateTime? RedeemBy { get; set; }

        [JsonProperty("redeem_by")]
        internal long? RedeemByInternal
        {
            get
            {
                if (!RedeemBy.HasValue) return null;

                return EpochTime.ConvertDateTimeToEpoch(RedeemBy.Value);
            }
        }
```

`TrialEnd` uses a string because they can handle the `"now"` value though

Reply:
Good points. I updated the test to use the same delay, and more importantly I updated the `DueDateInternal` variable to be a `long?` instead of a `String`.